app/build.gradle.kts

```diff
@@ -5,12 +5,12 @@ plugins {
 
 android {
     namespace = "me.devcexx.call2alarm"
-    compileSdk = 33
+    compileSdk = 34
 
     defaultConfig {
         applicationId = "me.devcexx.call2alarm"
-        minSdk = 33
-        targetSdk = 33
+        minSdk = 34
```

Comment:
My phone is still stuck at sdk level 33, and I'm probably the only user of this app. Can we keep the minSdk back in 33 so I can still using the app? :sweat_smile: